### PR TITLE
Add remove_polarization

### PR DESCRIPTION
**Pull request by irischang020**

### CODE REVIEW

File: pysingfel/detector/base.py [hunk 1]
```diff
@@ -311,7 +322,7 @@ def get_fxs_photons_slices(self, particles, beam_focus_radius, jet_radius, mesh_
                     field_acc += self.add_phase_shift(slices[i], coords[count])
                     count += 1
         return self.add_correction_and_quantization(np.square(np.abs(field_acc)))
-
+        #return np.random.poisson(np.multiply(np.square(np.abs(field_acc)), (self.linear_correction/self.polarization_correction)))
```
Comment (chuckie82) [inline pysingfel/detector/base.py]: remove

File: pysingfel/detector/base.py [hunk 2]
```diff
@@ -301,16 +301,14 @@ def get_fxs_photons_slices(self, particles, beam_focus_radius, jet_radius, mesh_
         mesh, voxel_length= self.get_reciprocal_mesh(voxel_number_1d = mesh_length)
         state, coords = distribute_particles(particles, beam_focus_radius, jet_radius)
         count = 0
+        field_acc = np.zeros((4, 512, 512), dtype=np.complex128)
```
Comment (chuckie82) [inline pysingfel/detector/base.py]: do not hardcode pixel dimensions for pnccd. this function needs to work for all detector dimensions

File: pysingfel/particlePlacement.py [hunk 3]
```diff
@@ -34,6 +28,7 @@ def distribute_particles(particles, beam_focus_radius, jet_radius):
     N = sum(particles.values()) # total number of particles
     coords = np.zeros((N,3)) # initialize N*3 array
     # generate N*3 random positions
+    np.random.seed(10)
```
Comment (chuckie82) [inline pysingfel/particlePlacement.py]: remove

File: examples/scripts/ExampleMultipleChaperones.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 # Load beam
 beam = ps.Beam(os.path.join(pwd,'../input/beam/amo86615.beam'))
 
-geom = os.path.join(pwd,'../input/lcls/amo86615/PNCCD::CalibV1/Camp.0:pnCCD.1/geometry/0-end.data')
```
Comment (chuckie82) [inline examples/scripts/ExampleMultipleChaperones.py]: this line is correct

Comment (irischang020) [reply examples/scripts/ExampleMultipleChaperones.py]: didn't work with this line

Comment (irischang020) [reply examples/scripts/ExampleMultipleChaperones.py]: geom = os.path.join(pwd,'../lcls/amo86615/PNCCD::CalibV1/Camp.0:pnCCD.1/geometry/0-end.data')

Comment (chuckie82) [reply examples/scripts/ExampleMultipleChaperones.py]: I cleaned up the input folder. you will need to do this:
cd pysingfel/examples/input  
Download a copy of lcls.tar.gz in the /input folder from https://stanford.box.com/s/e7c30tvhfz0485j2xr48rnrvel8a4yby
tar -xvf lcls.tar.gz

Please update README if this doesn't work

**Pull request merged**